src/app/room/room.cpp

```diff
@@ -1,14 +1,27 @@
 #include "room.hpp"
+#include "clinic_facade.hpp"
 
-Room::Room(const uint roomNumber, const std::vector<Treatment>& treatments, std::shared_ptr<Warehouse>& ptrToWarehouse)
-    : roomNumber_{roomNumber}, treatments_{treatments}, ptrToWarehouse_{ptrToWarehouse}
+Room::Room(const unsigned& room_id, const std::vector<Treatment>& treatments, std::shared_ptr<Warehouse>& warehouse)
+    : room_id_{room_id}, treatments_{treatments}, warehouse_{warehouse}, visits_{}
 {
 }
 
-uint Room::getRoomNumber() const
+void Room::addVisit(const std::shared_ptr<Visit>& visit)
+{
+    visits_.emplace_back(visit);
+}
```

Comment:
Functions that add associations should also make a backward connection to an other class association